### DISCUSSION TITLE
need Help here SEB Problem

### DISCUSSION THREAD

**Discussion by lay9423** [SafeExamBrowser/seb-win-refactoring#276] [Q&A]
so,i use my laptop and the only thing connected to my computer are my webcam. the thing is it keeps on saying the aactive display configuration is not permitted. 1 internals or 1 external things... can someone help me how to solve this
[2021-12-06_03h56m52s_Runtime.log](https://github.com/SafeExamBrowser/seb-win-refactoring/files/7656846/2021-12-06_03h56m52s_Runtime.log)
?

**Comment by dbuechel** [comment]
Please try the steps mentioned in #246.
